chaoscenter/web/src/utils/testUtils.tsx

```diff
@@ -2,39 +2,55 @@
 
 import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
+import { ApolloClient, ApolloProvider, InMemoryCache } from '@apollo/client';
 import { StringsContext } from '@strings';
 import strings from 'strings/strings.en.yaml';
 import '../bootstrap.scss';
 import { AppStoreContext } from '@context';
+import { ReactQueryProvider } from '@api/ReactQueryProvider';
 
 interface TestWrapperProps {
   children: React.ReactElement;
 }
 
+export interface APIConfig {
+  gqlEndpoints: {
+    chaosManagerUri: string;
+  };
+  restEndpoints: {
+    authUri: string;
+    chaosManagerUri: string;
+  };
+}
+
 export const findDialogContainer = (): HTMLElement | null => document.querySelector('.bp3-dialog');
 export const findPopoverContainer = (): HTMLElement | null => document.querySelector('.bp3-popover-content');
 
 export function TestWrapper({ children }: TestWrapperProps): React.ReactElement {
   const getString = (key: string): string => key;
-
+  const appoloClient = new ApolloClient({ cache: new InMemoryCache() });
   return (
-    <BrowserRouter>
-      <AppStoreContext.Provider
-        value={{
-          projectID: 'litmuschaos-test-project',
-          projectRole: 'Owner',
-          currentUserInfo: {
-            ID: 'uid',
-            username: 'admin',
-            userRole: 'admin'
-          },
-          renderUrl: `/account/uid`,
-          matchPath: '/account/:accountID',
-          updateAppStore: () => void 0
-        }}
-      >
-        <StringsContext.Provider value={{ data: strings as any, getString }}>{children}</StringsContext.Provider>
-      </AppStoreContext.Provider>
-    </BrowserRouter>
+    <ReactQueryProvider>
+      <ApolloProvider client={appoloClient}>
+        <BrowserRouter>
+          <AppStoreContext.Provider
```

Comment:
AppStoreContext Provider should be the top most in the order, here is the revised order for reference

```tsx
<AppStoreContext.Provider>
  <StringContext.Provider>
    <ReactQueryProvider>
       <ApolloProvider>
          <BrowserRouter />
       </ApolloProvider>
    </ReactQueryProvider>
  </StringContext.Provider>
</AppStoreContext.Provider>
```